cnd/src/http_api/routes.rs

```diff
@@ -28,6 +28,8 @@ use serde::Serialize;
 use std::collections::HashMap;
 use warp::{http, http::StatusCode, Rejection, Reply};
 
+const INVOICE_EXPIRY_SECS: u32 = 3600;
```

Comment:
If you make the constructor if `RelativeTime` a `const fn`, then you can initialize it directly as `RelativeTime` instead of `u32`.

Reply:
TIL I learn that we should have more `const fn` constructors.

Reply:
It is a mild guess but I think you should be able to `const fn` the ctor :)